examples/compress_file.c

```diff
@@ -30,6 +30,7 @@
 
 
 int main(int argc, char* argv[]) {
+  blosc_init();
```

Comment:
I'm curious about why the calls to `blosc_init` and `blosc_destroy`. I thought it is not necessary to do those calls. Perhaps my knowledge of it is old?

Reply:
I used the `blosc_init` to initialize a global variable for the io plugin, but it is removed in the last commit.

Although `blosc_init` and `blosc_destroy` are not necessary, I think that their usage should be recommendable (`blosc_init` initializes some structures):

https://github.com/Blosc/c-blosc2/blob/f489cdbd0a2567e07365671618fd859a31eec4ae/blosc/blosc2.c#L3241-L3252

@FrancescAlted ?

Reply:
Yes.  `blosc_init()` and `blosc_destroy()` were introduced initially in C-Blosc (1) so as to be able to initialize global structures.  C-Blosc2 uses contexts, so the usefulness of these is more limited now.  But we want to keep API compatibility with C-Blosc (1), so we need to keep them around in C-Blosc2 too.